app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -22,12 +21,35 @@ type ParamsType = {
 // Control what happens when a dynamic segment is visited that was not generated with generateStaticParams.
 export const dynamicParams = false;
 
+// Storyblok bridge options.
+const bridgeOptions = {
+  preventClicks: true,
+  resolveLinks: 'story',
+};
+
+/**
+ * Init on the server.
+ */
+storyblokInit({
+  accessToken: process.env.STORYBLOK_ACCESS_TOKEN, // Preview token because this is in server side.
+  use: [apiPlugin],
+  apiOptions: {
+    region: 'us',
+  },
+  components: Components,
+  enableFallbackComponent: true,
+  customFallbackComponent: ({blok}) => {
+    return (
+      <div className='rs-p-6 bg-red-600'><h2 className='text-white'>{blok.component} component is missing from the codebase.</h2><p className='text-white'>Source blok UID: {blok._uid}</p></div>
+    );
+  },
+});
```

Comment:
Storyblok needs to be initialized on the server side as well. Here, we can use the `preview` token as it doesn't get put into the front-end code.

I've also added the fallback component as this is the place they are doing it now.